https://github.com/Qiskit/qiskit-aer/pull/1741

From 5bf744a9fc033aee2eb448e2595f8e35734982e3 Mon Sep 17 00:00:00 2001
From: Sam James <sam@gentoo.org>
Date: Fri, 10 Mar 2023 04:56:34 +0000
Subject: [PATCH] Remove redundant wheel dep from pyproject.toml

Remove the redundant `wheel` dependency, as it is added by the backend automatically.

Listing it explicitly in the documentation was a historical mistake and has been fixed since,
see: [pypa/setuptools@f7d30a9](https://github.com/pypa/setuptools/commit/f7d30a9529378cf69054b5176249e5457aaf640a).
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 requires = [
   "setuptools",
-  "wheel",
   "conan<2.0.0",
   "scikit-build>=0.11.0",
   "cmake!=3.17.1,!=3.17.0",

